tests/conftest.py

```diff
@@ -82,6 +82,7 @@ def dwx_target():
         "user": os.getenv("DBT_HIVE_USER"),
         "password": os.getenv("DBT_HIVE_PASSWORD"),
         "http_path": os.getenv("DBT_HIVE_HTTP_PATH") or "cliservice",
+        "usage_tracking": False,
```

Comment:
Let's make it configurable such that we can configure the following options rather than boolean value
1. Send the events 
2. Log the events 
3. Completely skip (no log) the events. 

if we can parse these logs, then these event logs can be useful to construct the timeline and queries.